ledger/accountdb.go

```diff
@@ -193,11 +193,7 @@ type persistedOnlineAccountData struct {
 	rowid       int64
 	// the round number that is associated with the baseOnlineAccountData. This field is the corresponding one to the round field
 	// in persistedAccountData, and serves the same purpose.
-	round basics.Round
-}
-
-type wrappedPersistedOnlineAccountData struct {
-	data     persistedOnlineAccountData
+	round    basics.Round
 	updRound basics.Round
```

Comment:
please update lookupOnline method to set this field, and other places where persistedOnlineAccountData is instantiated.

Reply:
The only other places where persistedOnlineAccountData seem to be instantiated seem to be in tests and in Old functions and do not pass in a round either. Does updRound need to be set in those?

Reply:
for consistency, lookup and accountsLoadOld need to set it. Agree about tests.

Btw, I noticed this:
```
// before compares the round numbers of two persistedAccountData and determines if the current persistedAccountData
// happened before the other.
func (pac *persistedOnlineAccountData) before(other *persistedOnlineAccountData) bool {
	return pac.round < other.round
}
```

This is needed for LRU cache, could you confirm it is not used in the new cache you wrote?

Reply:
I do not use it in the new cache that is correct. I'm still a bit confused about the accountsLoadOld since I don't see anything there related to round numbers (the old one or the new one).

Reply:
I see, right. Lets move on then. We do not use oldAcct in any way after writing newAcct in to the DB.